server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/functions.js

```diff
@@ -507,20 +563,25 @@ function getProblemDetails() {
  * stores it on a sessionStorage variable
  */
 function getReplication() {
-  return $.getJSON('/rest/replication', function(data) {
-    sessionStorage.replication = JSON.stringify(data);
+  var call = '/rest/replication';
+  console.info("Retrieving " + call);
+  return $.getJSON(call, function(data) {
+    createRestGetLog(call, data, 'replication');
   });
 }
 
 //// Overview Plots Rest Calls
+//// Note: console.debug messages may not show by default in some browsers
```

Comment:
I think the logging level concept is pretty standard, so you probably don't need this comment here.

```suggestion
```

Or, you could clarify that the choice of using debug was intentional:
 
```suggestion
//// Note: console.debug avoids spamming the console by default in browsers that support it
```